### ISSUE TITLE
No longer works with guard 2.9

### ISSUE THREAD

**Issue by jf647** [Nordstrom/guard-foodcritic#7]
guard-foodcritic pulls in guard here:

https://github.com/cgriego/guard-foodcritic/blob/master/lib/guard/foodcritic.rb#L2

The file guard/guard.rb is no longer present in guard v2.9.0.  In v2.8.2 it was, but it contained deprecation warnings:

https://github.com/guard/guard/blob/v2.8.2/lib/guard/guard.rb#L12-22

Going back in the history, this deprecation has been documented in various forms for a while (Sept 2013):

https://github.com/guard/guard/blob/v2.0.1/lib/guard/guard.rb#L16-19

The solution for now is to pin guard to 2.8.2 in your Gemfile, but guard-foodcritic is in need of a refactor to work with the supported API.

Arguably the guard author should have held off on this for a major version bump (i.e. guard-foodcritic should have broken in the first 3.x release), but I'm not sure if they're following proper semver.

**Comment by flaccid** [comment]
+1
I pinned guard to 2.8.2 for it to work and observed:
```
19:10:52 - WARN - Starting with Guard 2.0, Guard::Foodcritic should inherit from Guard::Plugin
> [#] instead of Guard::Guard.
> [#] Please note that the constructor signature has changed from
> [#] Guard::Guard#initialize(watchers = [], options = {}) to
> [#] Guard::Plugin#initialize(options = {}).
> [#] For more information on how to upgrade for Guard 2.0, please head over
> [#] to: https://github.com/guard/guard/wiki/Upgrading-to-Guard-2.0#changes-in-guardguard
> [#]
> [#] Deprecation backtrace: /Users/chris.fordham/src/github/xhost-cookbooks/system/vendor/bundle/ruby/2.0.0/gems/guard-2.8.2/lib/guard/ui.rb:107:in `deprecation'
> [#] 	 >/Users/chris.fordham/src/github/xhost-cookbooks/system/vendor/bundle/ruby/2.0.0/gems/guard-2.8.2/lib/guard/guard.rb:94:in `initialize'
> [#] 	 >/Users/chris.fordham/src/github/xhost-cookbooks/system/vendor/bundle/ruby/2.0.0/gems/guard-foodcritic-1.0.3/lib/guard/foodcritic.rb:9:in `initialize'
```

**Comment by jf647** [comment]
Fixed in upcoming 1.1.0 release
